src/sass/_variables.scss

```diff
@@ -0,0 +1,19 @@
+//fonts
+$title-font: 'Bungee Shade', cursive;
+$pop-font: 'Bangers', cursive;
+$header-font: 'Bungee', cursive;
+$body-font: 'Bungee Hairline', cursive;
+
+//colors
+$light-green: #e5f4d3;
+$purple: #512fd9;
+$red-dark: #ff404a;
+$red-light: #ff3644;
+$yellow: #e6f51d;
+$brown: #553743;
+
+//colors dark mode:
+$dark-purple: #241c53;
+$teal: #03dac6;
+$dark-yellow: #eee8a9;
+$pop-purple: #ad7cfc;
```

Comment:
Thank you for doing this, so we don't need to refer to the Figma design constantly. :)

Reply:
love a variable!